db/lab_codes/ValueSet-qualitative-lab-result-findings.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
These shouldn't live in `db` because they don't have anything to do with the database. Maybe a new top level `resources` directory.

Reply:
resolved